### PR TITLE
LPS-71382 Allow administrators to configure OpenOffice using System Settings, allowing export and import of configurations

### PR DESCRIPTION
**Pull request by pei-jung**
journal changes sent to the journal sub-repo https://github.com/liferay/com-liferay-journal/pull/206

/cc @jorgeferrer

### CODE REVIEW

File: modules/apps/collaboration/document-library/document-library-document-conversion/src/main/java/com/liferay/document/library/document/conversion/internal/upgrade/v1_0_0/UpgradeOpenOfficeConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+		_configurationAdmin = configurationAdmin;
+		_prefsProps = prefsProps;
+	}
+
+	protected void doUpgrade() throws Exception {
+		Configuration configuration = _configurationAdmin.getConfiguration(
+			OpenOfficeConfiguration.class.getName(), StringPool.QUESTION);
+
+		Dictionary properties = configuration.getProperties();
```
Comment (drewbrokke) [inline modules/apps/collaboration/document-library/document-library-document-conversion/src/main/java/com/liferay/document/library/document/conversion/internal/upgrade/v1_0_0/UpgradeOpenOfficeConfiguration.java]: It's possible we don't need to do this.  If this upgrade is triggering, the configuration won't have any property values, and they will all be overridden anyways by the code below.

**Comment by drewbrokke** [comment]
Everything looks great! Nice work.  I made a single comment regarding the Upgrade, but it's not a major thing.

Also, it looks like we have some real test failures above.  Can you resolve those and re-send?

Thanks!

**Comment by drewbrokke** [comment]
I see - there are two DocumentConversionUtil.java files.  Are the `wiki-web` jsps using the correct one?

**Comment by pei-jung** [comment]
Yea, wiki-web is using the wrong one. I'll fix it.

**Comment by drewbrokke** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**